### ISSUE TITLE
output about test_result file

### ISSUE THREAD

**Issue by RowanLiu97** [lehaifeng/T-GCN#37]
Hi Mr Le
in paper T-GCN, about SZ-taxi dataset, This dataset consists of the taxi trajectory of Shenzhen from Jan. 1 to Jan. 31, 2015 and you aggregated the traffic speed on each road every 15 minutes.  80% of the data is used as the training set and the remaining 20% is used as the test set, that means Jan.27-Jan.31 is the prediction result. However, in test_result(result of prediction ) file, there are 591 rows that does not match Jan.27-Jan.31. In theory, it should be 576 rows. I have the same question on another data set.
I am looking forward to your reply Thank you!
